data/editor.config

```diff
@@ -287,8 +287,9 @@
         <include field="operator" />
         <include field="postcode" />
       </type>
-      <type id="amenity-parking" can_add="no">
+      <type id="amenity-parking">
         <include field="name" />
+        <include field="operator" />
```

Comment:
Opening hours?

Reply:
Access? Fee?

Reply:
We don't have access and fee fields support in the editor yet.